### ISSUE TITLE
Dynamic Popup opening with Tab breaks navigation to Braille Panel

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#29302]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1.  Ensure the Brailll window is enabled.
2. Navigate to any note on a score and press tab, notice focus shifts to the Braille window.
3. Navigate to a dynamic n the score and press Tab, notice the Dynamic Popup is opened.

This breaks navigation to the Braille panel when on a dynamic mark.  Tab should always jump to the next panel top level item.

However, I'm not sure if this is the way this was designed for all users to open the dynamic popup with Tab as pressing Enter or F2 seems to get a different result for me.

1.  With screen reader enabled navigate to a dynamic mark and press Tab.  Notice you can then arrow down through the dynamic options of ppppp to fffff, and then down to fp, sfz etc.  However, I couldn't get to cresc or hairpins if they are there and left/right arrows seemed very confusing to navigate. .
2Navigate to the place that says "other Dynamic Button" and pres enter, nothing seems to happen.  No more options seem to appear.
3. Press Esc to close and again navigate to a dynamic mark and this time press F2 or Enter.  Some sort of window opens but the screen reader is completely silent with all arrows, tabs or other movement, so unsure if this is the same window or something different?

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
Correction - F2 & Enter do seem to open this pop up also, it is just that the screen reader reports nothing with arrows, if I then Tab I am on it and it reads.  So, can focus move there directly when Enter or F2 are pressed or the blind user is left arrowing around wondering why nothing reads and not knowing this popup has opened.  This also occurs if you are on the Braille window and enter a hairpin via the Add menu, you get stuck in a non speaking pop up.

**Comment by jmwmusic** [comment]
Apologies for the additional comment.  I have just realised that this popup is probably popping up as soon as I alt arrow on to the dynamic, I was just unaware of it.  So Tab moves me to it.  The real issue then is that the order of Tab should be always from the score direct to the Braille panel, could this be a shift Tab to get back to the popup and if you press F2 or Enter on a dynamic focus automatically jumps to it?

**Comment by jrbowden** [comment]
Personally, I think Tab should never automatically open something - Tab is for navigation through items. So I believe it would be better to use F2 to open the Dynamics panel (or for that matter, edit slurs - same problem).
Thus: I would suggest:
1. Move to a dynamic in a score.
2. If you want to edit it, press F2 - just like any other text element.
3. If you press Tab at this point, focus should move to the next panel, just like for a note or other elements.

Note also that the navigation does not get stuck in the Dynamics panel if the Braille panel is not open.
Steps:
1. Ensure the braille panel is closed
2. In the score panel, navigate to a note.
3. Press Tab and you are on the Workspace toolbar (correct)
4. Now move to a dynamic in the score.
5. Press Tab and the Dynamics panel appears (current correct behaviour)
6. Press Tab again and you correctly move to the Workspace toolbar as above.

So behaviour is different depending on if the braille panel is shown or not.
